test_database.py

```diff
@@ -96,6 +101,11 @@ def test_hoenn():
 def test_sinnoh():
     region_test('sinnoh')
 
+def test_sinnoh():
```

Comment:
https://travis-ci.org/LazoCoder/Pokemon-Terminal/builds/265651400#L470 This should be `test_unova()` ?  If so then this is worth fixing.